### ISSUE TITLE
NODE_ENV is not a recognized internal or external command Solution

### ISSUE THREAD

**Issue by ssysm** [tmcw/docbox#86]
Hi,
just came across this issue when the application is build on a Windows box and the command telling me that `NODE_ENV is not a recognized internal or external command`

full log:
```
C:\Users\admin\IdeaProjects\spi\docs>npm run build

> API-Docs@1.1.1 build C:\Users\admin\IdeaProjects\spi\docs
> NODE_ENV=production browserify src/index.js | uglifyjs -c -m > bundle.js

'NODE_ENV' is not recognized as an internal or external command operable program or batch file.

npm ERR! code ELIFECYCLE
npm ERR! errno 255
npm ERR! API-Docs@1.1.1 build: `NODE_ENV=production browserify src/index.js | uglifyjs -c -m > bundle.js`
npm ERR! Exit status 255
npm ERR!
npm ERR! Failed at the API-Docs@1.1.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\admin\AppData\Roaming\npm-cache\_logs\2018-06-03T05_01_18_255Z-debug.log
```

This happend to me like three month ago, and I just removed `set NODE_ENV=production` part and build the application in development mode and today came across another package called `cross-env` that can let me build the application in production mode on Windows.

Would be nice if you can test it out and add to the dev dependencies or I can just pr one. 👍

**Comment by tmcw** [comment]
Sure, I'd accept a PR using `cross-env` to make that command compatible with non-posix shells

**Comment by tmcw** [comment]
Fixed in #87
